JavascriptWebAppSample/README.md

```diff
@@ -0,0 +1,45 @@
+# Javascirpt Web App Sample
+
+The [Azure Active Directory Authentication Library (ADAL)](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-authentication-libraries) enables client application developers to authenticate cloud users and obtain access tokens for API usage.
```

Comment:
The intro needs to describe the sample, not ADAL.  Set the stage for why this sample exists.  Look at your readme for the Device Profile sample.  Looking at some of your other readme files, you should go back and correct Managed Client and Client Library too